apluslms_roman/builder.py

```diff
@@ -51,24 +51,39 @@ def build(self, step_refs: list = None):
 
 class Engine:
     def __init__(self, backend_class=None, settings=None):
+        backend_name = None
         if backend_class is None:
             if settings and 'backend' in settings:
                 backend_class = settings['backend']
+                if 'backends' in settings and backend_class in settings['backends']:
+                    backend_name = backend_class
+                    backend_class = settings['backends'][backend_name]['type']
             else:
                 from .backends.docker import DockerBackend as backend_class
         if isinstance(backend_class, str):
             if '.' not in backend_class:
                 backend_class = BACKENDS.get(backend_class, backend_class)
-            backend_class = import_string(backend_class)
+            try:
+                backend_class = import_string(backend_class)
+            except ImportError as err:
+                err.backend = backend_class
```

Comment:
It would be better to raise a different error, so only it would be captured in cli and thus any other importerror would not be captured by the cli.

Reply:
hm well cli could also raise the error if it doesn't have the backend attribute?

Reply:
I think it's cleaner to use different error class, than adding a variable to it.